src/I18n/Number.php

```diff
@@ -263,6 +284,31 @@ public static function defaultCurrency($currency = null)
         return self::$_defaultCurrency;
     }
 
+    /**
+     * Getter/setter for default currency format
+     *
+     * @param null $currencyFormat Default currency format to be used by currency()
+     * if $currencyFormat argument is not provided. If boolean false is passed, it will clear the
+     * currently stored value
+     * @return string CurrencyFormat
+     */
+    public static function defaultCurrencyFormat($currencyFormat = null)
+    {
+        if (!empty($currencyFormat)) {
+            return self::$_defaultCurrencyFormat = $currencyFormat;
```

Comment:
`static::` in framework context.

Reply:
Again, just copied how `defaultCurrency` works. Should I clean all that up while I'm here?